ncm-opennebula/src/main/perl/opennebula.pm

```diff
@@ -1,42 +1,361 @@
+ e.g. by distributing the public key(s) of the OpenNebula host over the cluster.
+
+=item 3. Start OpenNebula services: C<< # for i in '' -econe -gate -novnc -occi -sunstone; do service opennebula$i stop; done >>
+
+=item 4. Run the component a first time.
+
+=item 5. The new oneadmin password will be available from C<< /var/lib/one/.one/one_auth >> file.
+The old auth files are stored with .quattor.backup extension.
+
+=item 6. It is also possible to change sunstone service password, just include
+'serveradmin' user and passwd within opennebula/users tree.
+In that case the component also updates the C<< sunstone_auth >> file.
+
+=back
+
+=head3 RESOURCES
+
+=head4 /software/components/${project.artifactId}
+
+The configuration information for the component.  Each field should
+be described in this section.
+
+=over 4
+
+=item * ssh_multiplex : boolean
+
+Set ssh multiplex options
+
+=item * cfg_group : string
+
+In some cases (such a Sunstone standalone conf with apache), some ONE conf files should be accessible by a different group (as apache).
+This variable sets the group name to change these files permissions.
+
+=item * host_hyp : string
+
+Set host hypervisor type
+
+=over 5
+
+=item * kvm
+
+Set KVM hypervisor
+
+=item * xen
+
+Set XEN hypervisor
+
+=back
+
+=item * host_ovs : boolean (optional)
+
+Includes the Open vSwitch network drives in your hypervisors. (OVS must be installed in each host)
+Open vSwitch replaces Linux bridges, Linux bridges must be disabled.
+More info: L<http://docs.opennebula.org/4.14/administration/networking/openvswitch.html>
+
+=item * tm_system_ds : string (optional)
+
+Set system datastore TM_MAD value (shared by default). Valid values:
+
+=over 5
+
+=item * shared
+
+The storage area for the system datastore is a shared directory across the hosts.
+
+=item * vmfs
+
+A specialized version of the shared one to use the vmfs file system.
+
+=item * ssh
+
+Uses a local storage area from each host for the system datastore.
+
+=back
+
+=back
+
+=head2 AII
+
+This document describes AII's OpenNebula hook.
+
+=head3 SYNOPSIS
+
+This AII hook generates the required resources and templates to instantiate/create/remove VMs within an OpenNebula infrastructure.
+
+=head3 RESOURCES
+
+=head4 AII setup
+
+Set OpenNebula endpoints RPC connector /etc/aii/opennebula.conf
+
+It must include at least one RPC endpoint and password.
+
+To set an https endpoint for example you can use:
+ url=https://host.example.com:2633
+
+By default ONE AII uses oneadmin user and port 2633.
+
+It is also possible to set a different endpoint for each VM domain or use a fqdn pattern
+as example:
+
+    [rpc]
+    password=
+    url=https://localhost/RPC2
+
+    [example.com]
+    password=
+    user=
+
+    [myhosts]
+    pattern=myhos\d+.example.com
+    password=
+    url=http://example.com:2633/RPC2
+
+=head4 configure
```

Comment:
@stdweird I'm not sure if this could be a bit confusing, we have `Configure`   for the server and also `aii_configure`  for the AII, should we use àii_configure`  instead? or just `configure`  for the AII?